packages/worker/package.json

```diff
@@ -19,5 +19,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "webpack": "5.88.2"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we need to add this dependency in the worker?

Reply:
If it's not declared, running `yarn build` from the `packages/worker` (like we do [here](https://github.com/DataDog/browser-sdk/blob/282e3c7aac9dbae47894a88d8e623418f036d2d3/scripts/lib/build-env.js#L43)) folder fails with `command not found: webpack`. It might be since the yarn upgrade. We may want to add it to other package if needed.